about.html

```diff
@@ -0,0 +1,92 @@
+        <section class="menu-logo menu-desktop">
+          <a href="index.html"><img src="img\logo.png" alt="logo"></a>
+        </section>
+        <ul class="item">
+          <li class="nav-item-btn"><a href="index.html">Home</a></li>
+          <li class="nav-item-btn"><a href="#logo">Logo</a></li>
+          <li class="nav-item-btn"><a href="#pastConcerts">Past Concerts</a></li>
+          <li class="nav-item-btn"><a href="#partnerSection">Partners</a></li>
+        </ul>
+      </section>
+    </nav>
+  </header>
+  <section class="intro">
+    <h2 class="h2-orange-title align-self-center">"Classical Music"</h2>
+    <h1 class="align-self-center">FREDERIC CHOPIN <br>CONCERT 2023</h1>
+    <section class="detail d-flex justify-content-center">
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+        Cum quidem doloremque in incidunt eum, quibusdam vitae 
+        beatae minus eius sunt perferendis aspernatur voluptas 
+        ipsa nesciunt, maxime fugit. Praesentium rerum unde 
+        doloribus labore voluptate quisquam ullam sapiente suscipit
+         numquam! Nobis, dicta iste? Doloremque vero sapiente 
+         obcaecati saepe labore nisi, dolores voluptatum!
+      </p>
+    </section>
+    <p class="location m-0">@ Warsaw, Poland</p>
+    <a class="text-dark" href="#"><p>chopinconcert@ccp.com</p></a>
+  </section>
+  <div id="logo" class="moveUp"></div>
+  <section class="logo dflex-column mb-5">
+    <h2 class="performers-and-pieces-title">Concerts Chopin Logo</h2>
+    <div class="bar"></div>
+    <p>Lorem ipsum dolor sit amet, consectetur 
+      adipisicing elit. Asperiores beatae laborum ea, 
+    </p>
+    <img class="px-5" src="img\logo.png" alt="logo">
+  </section>
+  <div id="pastConcerts" class="moveUp"></div>
+  <section class="past-concerts dflex-column">
+    <h2 class="performers-and-pieces-title">Past Concerts</h2>
+    <div class="bar"></div>
+    <p class="align-self-center">Lorem ipsum dolor sit amet, consectetur 
+      adipisicing elit. Asperiores beatae laborum ea, 
+    </p>
+    <section class="past-concerts-img dflex-column gap-3">
+      <section class="img-container dflex-column">
+        <h2>2020</h2>
+        <p>Chopin Concert Warsaw</p>
+      </section>
+      <section class="img-container dflex-column img2">
+        <h2>2018</h2>
+        <p>Chopin Concert Warsaw</p>
+      </section>
+    </section>
+  </section>
+  <section class="partner dflex-column" id="partnerSection">
+    <h2 class="performers-and-pieces-title" id="partner">Partner</h2>
+    <div class="bar"></div>
+    <ul class="d-flex">
+      <li>Kultura Dostepna</li>
+      <li>TFCI</li>
+      <li>Google</li>
+      <li>STEANWAY LYNGDORF</li>
+    </ul>
+  </section>
+  <footer class="d-flex align-items-center justify-content-center p-3 gap-3">
+    <img src="img\logo.png" alt="logo" width="250px">
+    <p>1830 Concert Chopin <br> Some Right Reserved.</p>
+  </footer>
```

Comment:
- According to the project requirements, the footer should have a dark background as seen in the design template. Please fix this 🙏 

Your project 👇:
<img width="944" alt="tbtrrt" src="https://user-images.githubusercontent.com/73607512/203841424-7e86d414-0d58-461e-9bb9-ba53b053461c.PNG">

Design template:
<img width="553" alt="gcgn" src="https://user-images.githubusercontent.com/73607512/203841801-3912d108-0bd5-477e-84f2-b69e63ec0ff8.PNG">